README.md

```diff
@@ -73,6 +106,13 @@ The generator installs your webpack files in the `client` folder. Foreman uses w
 
 Inside your Rails views, you can now use the `react_component` helper method provided by React on Rails.
 
+### Client-Side Rendering vs. Server-Side Rendering
+In most cases, you should use the provided helper method to render the React component from your Rails views with `prerender: false` (default behavior). In some cases, such as when SEO is vital or many users will not have JavaScript enabled, you can pass the `--server-rendering` option to the generator to configure your application for server-side rendering. Your JavaScript can then be first rendered on the server and passed to the client as HTML.
```

Comment:
In most cases, you should use the `prerender: false` (default behavior) with the  provided helper method to render the React component from your Rails views.